discussion/src/test/java/org/openedx/discussion/presentation/comments/DiscussionCommentsViewModelTest.kt

```diff
@@ -945,46 +945,6 @@ class DiscussionCommentsViewModelTest {
         assert(viewModel.uiState.value is DiscussionCommentsUIState.Success)
     }
 
-
-    @Test
```

Comment:
use-case?

Reply:
We no longer have the old `DiscussionCommentAdded` use case. So removed the TestCase as well.